README.md

```diff
@@ -87,3 +87,20 @@ vim.g.asyncrun_program = vim.empty_dict()
 vim.api.nvim_command("let g:asyncrun_program.vcvars64 = { opts -> '\"C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Auxiliary/Build/vcvars64.bat\" && ' .. opts.cmd }")
 vim.g.cmake_asyncrun_options = vim.tbl_extend('force', vim.g.cmake_asyncrun_options, {program = 'vcvars64'})
 ```
+
+## DAP configuration example
+
+You can customize the DAP configurations via `g:cmake_dap_configurations`:
+
+```lua
+vim.g.cmake_dap_configurations = {
+  type = "codelldb",
+  request = "launch",
+  host = "127.0.0.1",
+  port = 34567,
+  cwd = "${workspaceFolder}",
+  args = {},
```

Comment:
Could you remove these two lines? The user will be surprised with the behavior after the copying the example that overrides `cwd` and `args`.

```suggestion
```

Reply:
This example is based `CodeLLDB` settings, `CodeLLDB` would raise `Not implemented` internal error without these two lines, I think users would like to copy a working example.

Reply:
These two lines already defined by default by the extension.